docs/it/brand-identity.rst

```diff
@@ -0,0 +1,136 @@
+
+Il Trust Mark è l'elemento grafico ufficiale che dà prova agli Utenti dell’appartenenza al Sistema IT-Wallet degli attori del Sistema IT-Wallet e delle relative Soluzioni Tecniche con cui interagisce.  
+
+Di seguito i requisiti generali per il suo utilizzo, validi sia in riferimento a contesti di utilizzo fisici che digitali (e.g. siti web, applicazioni, documenti cartacei, materiali informativi stampati o video, etc.): 
+
+- il Trust Mark DEVE essere utilizzato esclusivamente per dare prova dell’appartenenza al Sistema IT-Wallet e non DEVE essere utilizzato per scopi diversi; 
+
+- il Trust Mark DEVE essere esposto esclusivamente dalle Soluzioni Tecniche che hanno concluso con successo il processo di certificazione; 
+
+- il Trust Mark DEVE essere quello presente all’interno delle Risorse Ufficiali e DEVE seguire le relative specifiche di utilizzo disponibili presso le Risorse Ufficiali, per garantirne un’adeguata visibilità in tutte le fasi dell’Esperienza Utente; 
+
+- il Trust Mark NON DEVE essere alterato, distorto, modificato o sostituito da loghi non ufficiali. 
+
+- il Trust Mark NON DEVE essere ridimensionato oltre i limiti minimi stabiliti dalle Risorse Ufficiali, in modo da garantire sempre una leggibilità ottimale su qualsiasi formato o dispositivo; 
+
+- il Trust Mark DEVE essere utilizzato garantendo l'area di rispetto minima definita nelle Risorse Ufficiali, al fine di garantirne un’adeguata visibilità e riconoscibilità. Altri elementi grafici o testuali NON DEVONO interferire con questa area di rispetto; 
+
+- il Trust Mark NON DEVE essere utilizzato su sfondi di colore che ne compromettano la visibilità o la leggibilità. DEVE essere garantito un contrasto adeguato tra il Logo e lo sfondo, in conformità con quanto definito nelle Risorse Ufficiali; 
+
+- il Trust Mark PUÒ essere associato a loghi, marchi o simboli di altri attori del sistema in accordo con le specifiche di coesistenza, in termini di proporzioni e visibilità, stabilite nelle Risorse Ufficiali. 
+
+Componenti 
+^^^^^^^^^^
+
+I componenti rappresentano quegli elementi del Sistema IT-Wallet che abilitano l’Utente a interagire con le diverse Soluzioni Tecniche tramite la propria Istanza IT-Wallet.  
+
+Le Risorse Ufficiali mettono a disposizione sia componenti complessi, ovvero template relativi a interi flussi, sia componenti atomici, ovvero singoli elementi da integrare all’interno di interfacce preesistenti (e.g. i Pulsanti di Ingaggio). 
+
+Di seguito i requisiti generali: 
+
+- gli Attori Primari DEVONO utilizzare esclusivamente le Risorse Ufficiali e DEVONO rispettare le indicazioni descritte nelle Risorse Ufficiali; 
+
+- gli Attori Primari POSSONO scegliere quale configurazione, tra quelle rese disponibili, implementare, ma DEVONO comunque garantire il corretto utilizzo dei componenti atomici come i Pulsanti di Ingaggio; 
+
+- gli Attori Primari DEVONO garantire il costante aggiornamento dei componenti, in linea con l’ultima versione resa disponibile. 
```

Comment:
```suggestion
- gli Attori Primari DEVONO garantire il costante aggiornamento dei componenti, in linea con l’ultima versione resa disponibile. 
```

qui bisogna dire in che tempi

non è possibile pensare che all'immediata emissione di una nuova linea guida, tutti possano adeguarsi immediatamente. Bisogna normare dei tempi.

Reply:
idealmente, un adeguamento tecnico di media complessità può richiedere fino a 90 giorni